src/canvasPanelComponents/slideshow/src/components/SwappableViewer/SwappableViewer.tsx

```diff
@@ -165,26 +161,18 @@ const SwappableViewer: React.FC<SwappableViewerProps> = ({
           />
         </>
       ) : (
-        <SingleTileSource
-          manifest={manifest}
-          canvas={canvas}
-          notifyVideo={(bool: boolean, uri: string) => {
-            setVideo(bool);
-            setVideoUri(uri);
-          }}
-        >
-          <FullscreenButton {...fullscreenProps} />
-          {isInteractive ? (
-            <ZoomButtons
-              onZoomOut={determineIsZoomedOut() ? null : zoomOut}
-              onZoomIn={isZoomedIn() ? null : zoomIn}
-            />
-          ) : (
-            <></>
-          )}
-          {video && videoUri ? (
-            <IFrameYouTube url={videoUri} />
-          ) : (
+        <>
+          <YoutubeVideoSource />
+          <SingleTileSource manifest={manifest} canvas={canvas}>
+            <FullscreenButton {...fullscreenProps} />
+            {isInteractive ? (
+              <ZoomButtons
+                onZoomOut={determineIsZoomedOut() ? null : zoomOut}
+                onZoomIn={isZoomedIn() ? null : zoomIn}
+              />
+            ) : (
+              <></>
+            )}
```

Comment:
@ionab I've made a slight change to the video vs image. If the `SingleTileSource` can't find anything it will not return anything. We can use this (for our case) and just have a different component that will, if a video exists, render the youtube iframe. Let me know what you think.